augly/video/utils.py

```diff
@@ -35,6 +36,18 @@ def apply_ffmpeg_augmenter(
         shutil.move(result_path, output_path)
 
 
+def apply_ffmpeg_vidaug_augmenter(
+    augmenter: af.BaseFFMPEGAugmenter, video_path: str, output_path: Optional[str]
+) -> None:
+    video_path, output_path = helpers.validate_input_and_output_paths(
+        video_path, output_path
+    )
+
+    writer = WriteGear(output_filename=video_path, logging=True)
+    writer.execute_ffmpeg_cmd(augmenter.add_augmenter())
+    writer.close()
```

Comment:
somewhere in here inject the video_path and output_path into the returned command :)